src/main.cpp

```diff
@@ -130,20 +128,19 @@ int main(int argc, char** argv)
     }
     
     if (config()->get("OpenPreviousDatabasesOnStartup").toBool()) {
-        const QStringList filenames = config()->get("LastOpenedDatabases").toStringList();
-        for (int ii = filenames.size()-1; ii >= 0; ii--) {
-            QString filename = filenames.at(ii);
+        const QStringList fileNames = config()->get("LastOpenedDatabases").toStringList();
+        for (const QString &filename: fileNames) {
```

Comment:
Use asConst()

Reply:
asConst() function not needed with constant containers.

 [Qt A Porting Guide to C++11 Ranged For-Loops - Avoiding Detaching](https://www.kdab.com/goodbye-q_foreach/)

Reply:
Ummm... I was pretty sure it was non-const when I wrote that comment, but I must have overlooked it.